reV/config/sam_config.py

```diff
@@ -127,25 +127,33 @@ def downscale(self):
 
         Returns
         -------
-        str | None
+        dict | None
             Option for NSRDB resource downscaling to higher temporal
-            resolution. Expects a string in the Pandas frequency format,
-            e.g. '5min'.
+            resolution. The config expects a str entry in the Pandas
+            frequency format, e.g. '5min' or a dict of downscaling kwargs
+            such as {'frequency': '5min', 'variability_kwargs':
+            {'var_frac': 0.05, 'distribution': 'uniform'}}.
+            A str entry will be converted to a kwarg dict for the output
+            of this property e.g. '5min' -> {'frequency': '5min'}
         """
         if self._downscale is None:
-            downscale = []
+            ds_list = []
             for v in self.inputs.values():
-                downscale.append(v.get('downscale', None))
+                ds_list.append(v.get('downscale', None))
 
-            self._downscale = list(set(downscale))
+            self._downscale = ds_list[0]
```

Comment:
@MRossol, ds_list is just a temporary variable that gets the list of all the downscaling arguments for all SAM input json files and makes sure they're the same. self._downscale is the attribute that gets used outside of the property and is set here.